splinterd/Cargo.toml

```diff
@@ -53,8 +53,8 @@ toml = "0.5"
 [dev-dependencies]
 openssl = { version = "0.10" }
 reqwest = { version = "0.11", features = ["blocking"] }
-sabre-sdk = "0.8"
-transact = "0.4"
+sabre-sdk = "0.9"
```

Comment:
Does this still result in the sawtooth SDK being pulled in?

Reply:
Checking the lock file, the sawtooth SDK is still pulled in by transact